plugins/woocommerce/src/Blocks/BlockTypes/ProductStockIndicator.php

```diff
@@ -82,15 +83,13 @@ protected function render( $attributes, $content, $block ) {
 			return '';
 		}
 
-		$availability = $product->get_availability();
+		$availability = ProductAvailabilityUtils::get_product_availability( $product );
 
 		if ( empty( $availability['availability'] ) ) {
 			return '';
 		}
 
-		$low_stock_amount   = $product->get_low_stock_amount();
 		$total_stock        = $product->get_stock_quantity();
-		$is_low_stock       = $low_stock_amount && $total_stock <= $low_stock_amount;
```

Comment:
We don't use these two variables (`$low_stock_amount` and `$is_low_stock`) anymore.